index.html

```diff
@@ -126,98 +126,8 @@
                 <span class="line"></span>
             </p>
 
-            <div class="speakers">
+            <div class="speakers" id="speakers">
```

Comment:
- [ ] Good work with your about page but I think you can work more on it and make the design stick to the original design. Please Make sure that your about page section is similar to the original design you can see some differences in height, font size, and color for your paragraph, text, and titles. Please fix it  :+1: 

![Screenshot from 2022-10-19 19-59-42](https://user-images.githubusercontent.com/6615394/196792451-aaefc746-ec06-4723-a41e-391fee94b542.png)
![Screenshot from 2022-10-19 19-59-52](https://user-images.githubusercontent.com/6615394/196792972-b47bb5be-b55f-4341-bc59-7c4025c3a1c9.png)